akka-docs/src/main/paradox/io-udp.md

```diff
@@ -1,3 +1,6 @@
+---
+project.description: Using UDP with Akka.
+---
```

Comment:
Kind of the same as for TCP but here we don't have a stream API.

Maybe "Low level API for using UDP with classic actors"?